docs/protocol/mint_packages_nfts.md

```diff
@@ -36,7 +36,7 @@ In order to mint a software package as an NFT, you must ensure that you have:
 
 ## Mint a component
 
-Connect your wallet to the Protocol app, open the [components section](https://protocol.autonolas.network/components) and press the _Mint_ button. Fill in the data of your component:
+Connect your wallet to the {{ autonolas_protocol_registry_dapp }}, open the [components section](https://registry.olas.network/components) and press the _Mint_ button. Fill in the data of your component:
```

Comment:
Is there an option like:

```
[components section]({{ autonolas_protocol_registry_dapp }}/components)
```
?

If yes, then also `components` and `agents` sections can be replaced below.

Reply:
Done